[cross-Microsoft] (Power Query) When to use Power Query or Office Scripts

AlexJerabek opened the pull request:
This article adds to the series that try to guide users to the correct automation platform. It focuses on Power Query versus Office Scripts and highlights the strengths of both.

opbld32 commented:
Docs Build status updates of commit _[8cc8540](https://github.com/OfficeDev/office-scripts-docs/commits/8cc854018a4cdd9564ad580c922f6e5bf4cc0982)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C5%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111051959264818-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-06T20%3A00%3A06Z&sr=b&sp=r&sig=MY70Sv3PSATwvQPRUgVWGgXnYvs0O6QOmlddy692woI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[b2799aa](https://github.com/OfficeDev/office-scripts-docs/commits/b2799aacabdf218b40b565caca7c1f6eac304b43)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C5%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111052001500359-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-06T20%3A02%3A25Z&sr=b&sp=r&sig=Ad0amhCoPbgbKgIx96mdDERRoL6fSeVdILLz2lKqNeI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

AlexJerabek commented:
@jedunn had some great offline feedback around the transformation and retrieval aspects of PQ and the control over tables, charts, shapes, etc in OS. I'll incorporate that feedback and send a new version out soon.

peronald reviewed (changes requested):
So templates can be deployed to an organization. I think I found some documentation saying how to but it isn't obvious for how teams would do that or groups smaller than the org. Maybe we should just say something like Office Scripts allows for more granular permissioning and sharing

opbld32 commented:
Docs Build status updates of commit _[304ed9b](https://github.com/OfficeDev/office-scripts-docs/commits/304ed9b380824ddb57fc71e7cac674241782cd70)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) | [Details](#user-content-f45e42656cbeef15836566175e052ccf-8c1b4ed7)
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

<a id="f45e42656cbeef15836566175e052ccf-8c1b4ed7"></a>
### [docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md)
 - **Line 12, Column 262**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../develop/power-automate-integrations.md'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111091844337418-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-10T18%3A45%3A15Z&sr=b&sp=r&sig=YBSkPkGnJEb8DDz3DqRpSgC%2F59V%2BQCwLOn1ucfXYPKg%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[05317a5](https://github.com/OfficeDev/office-scripts-docs/commits/05317a56ccc2ccf3a77bcd438dceb199420577c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111091846101849-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-10T18%3A46%3A49Z&sr=b&sp=r&sig=f3NyZIgpNgSpCp8DzB2tjEIFsjp0gtKkEWGV1Gndpjg%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

AlexJerabek commented:
Thanks for the reviews @jedunn and @peronald. I updated the section on programmability to talk about visualizations and programming controls. I also removed the sharing section, since it seems like [PQ offers the same options](https://support.microsoft.com/en-us/office/view-and-update-a-shared-query-power-query-ece958c5-baf2-4789-9312-bb113e679e91?ocmsassetid=ha104079184&correlationid=70b4ddb2-9325-46b7-b274-c80f43e6711a&ui=en-us&rs=en-us&ad=us) we do (I added some nuance to the template section).

AlexJerabek commented:
Received offline approval from @peronald. @jedunn, mind taking another look?

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+We recommend Office Scripts when your needs go beyond data importing and transformation.
+
+Nearly everything you can do manually through the Excel UI is doable with Office Scripts. They are great for applying consistent formatting to workbooks. They create charts, PivotTables, shapes, images, and other worksheet visualizations. When you're  
```
There's an incomplete sentence at the end of this paragraph.

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+The inclusion of TypeScript code gives you a high degree of customization. Control logic like `if...else` statements makes your script robust. This lets you do things like conditionally read data without relying on complex Excel formulas or scan the workbook for unexpected changes before changing the workbook.
+
+Formatting can be applied with Power Query through [templates](https://templates.office.com/power-query-tutorial-tm11414620). However, templates are updated at the individual or organization level, whereas Office Scripts offer more granular access control.
```
```suggestion
Formatting can be applied with Power Query through [templates](https://templates.office.com/power-query-tutorial-tm11414620). However, Power Query templates are updated at the individual or organization level, whereas Office Scripts offer more granular access control.
```

AlexJerabek replied:
It's actually through Excel templates. I'll update the sentence to make that more clear.

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+## Platform dependencies
+
+Office Scripts is currently only for Excel on the web. Power Query is currently only for Excel on Desktop. Both can be used through Power Automate, which lets the flow work with Excel workbooks stored in OneDrive.
```
```suggestion
Office Scripts is currently only available for Excel on the web. Power Query is currently only available for Excel on Desktop. Both can be used through Power Automate, which lets the flow work with Excel workbooks stored in OneDrive.
```

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+Nearly everything you can do manually through the Excel UI is doable with Office Scripts. They are great for applying consistent formatting to workbooks. They create charts, PivotTables, shapes, images, and other worksheet visualizations. When you're  
+
+The inclusion of TypeScript code gives you a high degree of customization. Control logic like `if...else` statements makes your script robust. This lets you do things like conditionally read data without relying on complex Excel formulas or scan the workbook for unexpected changes before changing the workbook.
```
```suggestion
The inclusion of TypeScript code gives you a high degree of customization. Programmatic control logic like `if...else` statements makes your script robust. This lets you do things like conditionally read data without relying on complex Excel formulas, or scan the workbook for unexpected changes before changing the workbook.
```

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+Office Scripts offer more options for Power Automate integration. Scripts are tailored to your solutions. You define the [input and output of the script](../develop/power-automate-integration.md#data-transfer-in-flows-for-scripts), so it works with any other connector or data in the flow.
+
+:::image type="content" source="../images/scenario-task-reminders-last-flow-step.png" alt-text="The Excel Online (Business) connector in the flow designer showing the Office Scripts option.":::
```
For accessibility, I recommend adding a sentence describing what this screenshot illustrates. It's not immediately apparent to me. Something like, "The following screenshot shows an example of integrating..."

alison-mk commented on docs/resources/power-query-differences.md:
```diff
@@ -0,0 +1,52 @@
+
+Power Query is used in the [SQL Server](https://powerquery.microsoft.com/flow/) Power Automate connector. The [Transform data using Power Query](/connectors/sql/#transform-data-using-power-query) action lets you build a query in Power Automate. While this is a powerful tool for use with SQL Server, it does limit Power Query to that input source.
+
+:::image type="content" source="../images/power-query-flow-option.png" alt-text="The SQL Server connector in the flow designer showing the Power Query option.":::
```
I recommend adding a sentence describing this screenshot, too.

alison-mk reviewed (changes requested):
The only blocking issue is the incomplete sentence. Otherwise, I've left a handful suggestions that I think improve clarity and accessibility.

opbld32 commented:
Docs Build status updates of commit _[d6562a5](https://github.com/OfficeDev/office-scripts-docs/commits/d6562a5912ee800efb6f3a59feef0082cf4b6eb8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C23%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111232156310161-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-24T21%3A57%3A07Z&sr=b&sp=r&sig=pKC5%2FzbXy7cwDz83F3OPX3LjElNmfG3JYjY81F3ZCMI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[64bc8cc](https://github.com/OfficeDev/office-scripts-docs/commits/64bc8cc0a4561d93bb9cf2187f9da0b238b05985)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C23%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111232157186506-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-24T21%3A57%3A57Z&sr=b&sp=r&sig=ikVHJVbZ3BAgxOXCAgMyH%2FxfYvDJmiXcBBpM2X1GFuM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[5d02aef](https://github.com/OfficeDev/office-scripts-docs/commits/5d02aef102928801f06f152256abc5b53230d1b8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C23%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111232323334827-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-24T23%3A24%3A23Z&sr=b&sp=r&sig=R5hRIQQvY0jw4WroIQ1%2FPn%2FYlvxqXCcyy%2BI%2FX5fueCQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[693c590](https://github.com/OfficeDev/office-scripts-docs/commits/693c5908e3e1b9d9b0df19c41caa39699e625988)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/power-query-flow-option.PNG](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/images/power-query-flow-option.PNG) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/resources/power-query-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/resources/power-query-differences.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PQ/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/power-query-differences?branch=pr-en-us-425) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C23%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202111232325436012-425%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-24T23%3A26%3A23Z&sr=b&sp=r&sig=jZMH3yCwlsyS5l6hunAxkiWa6pSETSShfeXP2yL2eOs%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

alison-mk reviewed (approved):
Looks good!

lindalu-MSFT commented:
I'm so sorry I didn't see this one, but it looks like you all have it under control!

From: Alison McKay ***@***.***>
Sent: Tuesday, November 23, 2021 4:01 PM
To: OfficeDev/office-scripts-docs ***@***.***>
Cc: Linda Lu Cannon ***@***.***>; Review requested ***@***.***>
Subject: Re: [OfficeDev/office-scripts-docs] [cross-Microsoft] (Power Query) When to use Power Query or Office Scripts (PR #425)


@alison-mk approved this pull request.

Looks good!

-
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOfficeDev%2Foffice-scripts-docs%2Fpull%2F425%23pullrequestreview-814303721&data=04%7C01%7CLinda.Cannon%40microsoft.com%7C6a857bbe8a03418f56ff08d9aedd83fb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637733088707958860%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=uKnjArdNEPeMNWb%2BNC%2BTcwUB1Mirit2vS3mEw716k7g%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHKC5MYW7HF6YRRJRGVASU3UNQTMFANCNFSM5HOS5HEA&data=04%7C01%7CLinda.Cannon%40microsoft.com%7C6a857bbe8a03418f56ff08d9aedd83fb%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637733088707958860%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=kzQF40qrgwng88kdkdsTTpLVGmsbL1K3xnylgrdI7u0%3D&reserved=0>.
